src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java

```diff
@@ -34,6 +34,7 @@
 
     private Inventory inventory;
     private int size;
+    private final static String backpacks = "backpacks.";
```

Comment:
Constants should follow the snake-case convention and should be put on top.
Also, this constant's name is a bit meaningless towards what it does, something like `CONFIG_PREFIX` would be more accurate in my opinion.

Here's my suggested change, could you please also move it to the top of the class?

```suggestion
    private static final String CONFIG_PREFIX = "backpacks.";
```

Reply:
ff83a0183